virtual_ship/expedition/input_data.py

```diff
@@ -0,0 +1,176 @@
+        }
+        variables = {"U": "uo", "V": "vo", "S": "so", "T": "thetao"}
+        dimensions = {
+            "lon": "longitude",
+            "lat": "latitude",
+            "time": "time",
+            "depth": "depth",
+        }
+
+        # create the fieldset and set interpolation methods
+        fieldset = FieldSet.from_netcdf(
+            filenames, variables, dimensions, allow_time_extrapolation=True
+        )
+        fieldset.T.interp_method = "linear_invdist_land_tracer"
+        fieldset.S.interp_method = "linear_invdist_land_tracer"
+
+        # make depth negative
+        for g in fieldset.gridset.grids:
+            g.depth = -g.depth
+
+        # add bathymetry data
+        bathymetry_file = directory.joinpath("bathymetry.nc")
+        bathymetry_variables = ("bathymetry", "deptho")
+        bathymetry_dimensions = {"lon": "longitude", "lat": "latitude"}
+        bathymetry_field = Field.from_netcdf(
+            bathymetry_file, bathymetry_variables, bathymetry_dimensions
+        )
+        # make depth negative
+        bathymetry_field.data = -bathymetry_field.data
+        fieldset.add_field(bathymetry_field)
+
+        # read in data already
+        fieldset.computeTimeChunk(0, 1)
+
+        return fieldset
+
+    @classmethod
+    def _load_drifter_fieldset(cls, directory: str | Path) -> FieldSet:
+        filenames = {
+            "U": directory.joinpath("drifter_uv.nc"),
+            "V": directory.joinpath("drifter_uv.nc"),
+            "T": directory.joinpath("drifter_t.nc"),
+        }
+        variables = {"U": "uo", "V": "vo", "T": "thetao"}
+        dimensions = {
+            "lon": "longitude",
+            "lat": "latitude",
+            "time": "time",
+            "depth": "depth",
+        }
+
+        fieldset = FieldSet.from_netcdf(
+            filenames, variables, dimensions, allow_time_extrapolation=False
+        )
+        fieldset.T.interp_method = "linear_invdist_land_tracer"
+
+        # make depth negative
+        for g in fieldset.gridset.grids:
+            g.depth = -g.depth
+
+        # read in data already
+        fieldset.computeTimeChunk(0, 1)
+
+        return fieldset
+
+    @classmethod
+    def _load_argo_float_fieldset(cls, directory: str | Path) -> FieldSet:
+        filenames = {
+            "U": directory.joinpath("argo_float_uv.nc"),
+            "V": directory.joinpath("argo_float_uv.nc"),
+            "S": directory.joinpath("argo_float_s.nc"),
+            "T": directory.joinpath("argo_float_t.nc"),
+        }
+        variables = {"U": "uo", "V": "vo", "S": "so", "T": "thetao"}
+        dimensions = {
+            "lon": "longitude",
+            "lat": "latitude",
+            "time": "time",
+            "depth": "depth",
+        }
+
+        fieldset = FieldSet.from_netcdf(
+            filenames, variables, dimensions, allow_time_extrapolation=False
+        )
+        fieldset.T.interp_method = "linear_invdist_land_tracer"
```

Comment:
Should fieldset.S not also have the same interp_method?
```suggestion
        fieldset.T.interp_method = "linear_invdist_land_tracer"
        fieldset.S.interp_method = "linear_invdist_land_tracer"
```